### DISCUSSION TITLE
use ssh direct as a vpn in webssh?

### DISCUSSION THREAD

**Discussion by Aronablade** [isontheline/pro.webssh.net#889] [Q&A]
hi i have a vps that i use ssh direct to it as a vpn, i use the napsternetV, in napster i only need go to config and add a ssh config and enter my vps domain and ssh port (22) and my server username and pass and im ready to go after napsternet is connected my device and apps all go throw this connection lile a vpn,
i saw tunnelin webssh and an option vpn, but i couldnt make it work, it connects but my traffic doesnt go throw this tunnel, can you tell me how can i do it? how can i make allmy device use webssh tunnel?

**Comment by isontheline** [comment]
Hi @Aronablade 👋

Yet WebSSH doesn't forward trafic of your device.

I need to work on this issue before : https://github.com/isontheline/pro.webssh.net/issues/104
